front/pages/api/w/[wId]/webhook_sources/views/[viewId]/index.ts

```diff
@@ -14,7 +15,15 @@ import { apiError } from "@app/logger/withlogging";
 import type { Result, WithAPIErrorResponse } from "@app/types";
 import { assertNever, Err, Ok } from "@app/types";
 import type { WebhookSourceViewType } from "@app/types/triggers/webhooks";
-import { patchWebhookSourceViewBodySchema } from "@app/types/triggers/webhooks";
+
+const PatchWebhookSourceViewBodySchema = z.object({
+  name: z.string().min(1, "Name is required."),
+  description: z
+    .string()
+    .max(4000, "Description must be at most 4000 characters.")
+    .optional(),
```

Comment:
wondering if we should have description non optional, so we can precise display it in the AB and people using the trigger can know what they sub to

not urgent whatsoever

another way is also to determine a useful description via the webhook source provider + subscribed events + oauth details (ex : repo on GitHub)